src/main/java/com/github/kagkarlsson/scheduler/Scheduler.java

```diff
@@ -32,12 +32,11 @@
 
 import static com.github.kagkarlsson.scheduler.ExecutorUtils.defaultThreadFactoryWithPrefix;
 
-public class Scheduler implements SchedulerClient {
+public class Scheduler extends SchedulerClient.StandardSchedulerClient {
 
```

Comment:
What is the reason SchedulerClient interface was changed for extends here? The idea behind SchedulerClient is that you are able to "downgrade" the full Scheduler object to only a SchedulerClient, not exposing everything